queries/kotlin/highlights.scm

```diff
@@ -0,0 +1,162 @@
+;;; Kotlin 
+;; Issues
+
+; mapOf with type definition AND multiple entries are not highligted correctly. Issue in parser. Eg.
+; a = mapOf<String, String>(AA.value to AA.value, AA.value to AA.value)
+
```

Comment:
You could add a `@type` highlighting for the typical upperclass naming convention. Like in C++/Java/Python

Reply:
also
```
(type_identifier) @type
```

Reply:
annotations are actually superclass of type_identifier, so I needed to be bit more specific